include/csp/arch/csp_thread.h

```diff
@@ -90,6 +90,9 @@ typedef csp_thread_return_t (* csp_thread_func_t)(void *) __attribute__((stdcall
 */
 #if (CSP_FREERTOS)
 
+#include <FreeRTOS.h>
+#include <task.h>
```

Comment:
This should not be needed. The idea of /arch is specifically not to include platform specific headers

Reply:
There's a `#include <Windows.h>` a couple of lines further up, so that's a pretty moot point?
Please add a comment explaining this for future developers then :)

Reply:
Yeah, I don't have anything to do with the windows port. So I also don't see why that was needed there.